src/core/Global.ttslua

```diff
@@ -645,6 +646,20 @@ function emptyChaosBag()
   end
 end
 
+-- move all regular chaos tokens that are on the table to the chaos bag (e.g. sealed on cards)
+function findAllChaosTokens()
```

Comment:
The name here is unclear - find() implies no mutation, just a lookup.  Maybe returnAllChaosTokens()?

Reply:
Name is changed now